libnavui-dropin/api/current.txt

```diff
@@ -0,0 +1,563 @@
+    method public boolean getGranted();
+    property public final boolean granted;
+  }
+
+  public final class TripSessionStarterState {
+    method public boolean component1();
+    method public boolean component2();
+    method public com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState copy(boolean isLocationPermissionGranted, boolean isReplayEnabled);
+    method public boolean isLocationPermissionGranted();
+    method public boolean isReplayEnabled();
+    property public final boolean isLocationPermissionGranted;
+    property public final boolean isReplayEnabled;
+  }
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class TripSessionStarterViewModel extends com.mapbox.navigation.dropin.lifecycle.UIViewModel<com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState,com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction> {
+    ctor public TripSessionStarterViewModel(com.mapbox.navigation.dropin.component.navigation.NavigationStateViewModel navigationStateViewModel, com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState initialState = TripSessionStarterState());
+    method public com.mapbox.navigation.dropin.component.navigation.NavigationStateViewModel getNavigationStateViewModel();
+    method public com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState process(com.mapbox.navigation.core.MapboxNavigation mapboxNavigation, com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterState state, com.mapbox.navigation.dropin.component.tripsession.TripSessionStarterAction action);
+    property public final com.mapbox.navigation.dropin.component.navigation.NavigationStateViewModel navigationStateViewModel;
+  }
+
+}
+
+package com.mapbox.navigation.dropin.internal.extensions {
+
+  public final class AttributeSetEx {
+    method public static String navigationViewAccessToken(android.util.AttributeSet?, android.content.Context context);
+  }
+
+  public final class ContextEx {
+  }
+
+  public final class MapboxLifecycleExtensionsKt {
+    method @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public static <T extends com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver> androidx.lifecycle.LifecycleOwner attachCreated(androidx.lifecycle.LifecycleOwner, T... observers);
+    method @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public static <T extends com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver> androidx.lifecycle.LifecycleOwner attachResumed(androidx.lifecycle.LifecycleOwner, T... observers);
+    method @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public static <T extends com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver> androidx.lifecycle.LifecycleOwner attachStarted(androidx.lifecycle.LifecycleOwner, T... observers);
+  }
+
+  public final class MapboxNavigationEx {
```

Comment:
Could this be internal? Or will it require more changes?

Reply:
We already moved it to `internal` package. This should not show up in this file.

Reply:
Same as https://github.com/mapbox/mapbox-navigation-android/pull/5703#discussion_r849803637. Resolving this comment so we can track it in one place.